pkg/osutil/osutil.go

```diff
@@ -155,3 +155,13 @@ func WriteFile(filename string, data []byte) error {
 func WriteExecFile(filename string, data []byte) error {
 	return ioutil.WriteFile(filename, data, DefaultExecPerm)
 }
+
+// Return all files in a directory.
+func ListDir(dir string) ([]string, error) {
```

Comment:
I withdraw this. This version is moved from manager and it is faster than ioutil.ReadDir.